python/sglang/srt/constrained/xgrammar_backend.py

```diff
@@ -112,25 +109,16 @@ def __init__(
     ):
         super().__init__()
 
-        if import_error:
-            logger.warning(
-                f"Ignore import error for the grammar backend: {import_error}"
-            )
-            self.grammar_cache = None
-            return
-
         tokenizer_info = TokenizerInfo.from_huggingface(tokenizer)
```

Comment:
Assign vocab_size and stop_token_ids (from the chat_template, optionally but will make it more robust) when constructing tokenizer_info

Reply:
Agreed.. I had an in issue initially where the vocab_size provided by the tokenizer differed from `vocab_size` param provided when initialing the grammar backend. In my most recent commit I use `vocab_size` when creating `tokenizer_info`. My blocker now is that relating to GrammarMatcher accepting the stop token, but still trying to find the next token mask. I believe this could be related to the backend not having the correct stop tokens.

What is best way to access the chat templates's stop_token_ids information when initializing the grammar backend?